app/models/attribute/GlobalAttributeTable.scala

```diff
@@ -86,6 +89,13 @@ case class GlobalAttributeWithLabelForAPI(globalAttributeId: Int,
       )
     )
   }
+  val attributesToArray = Array("" + globalAttributeId, labelType, attributeSeverity, "" + attributeTemporary,
```

Comment:
Instead of doing the `"" + x` method, can you use `x.toString` everywhere? Just for the sake of code clarity.